zun/container/oci.py

```diff
@@ -16,10 +16,14 @@
 
 
 def merge_oci_runtime_config(parent, *cfgs):
+    cap_sets = ["bounding", "permitted", "inheritable", "effective", "ambient"]
```

Comment:
Probably worth pulling out as a constant, and citing https://man7.org/linux/man-pages/man7/capabilities.7.html